### ISSUE TITLE
ReadJournal query "currentPersistenceIds" with Casbah leads to MongoCommandException if response is > 16MB

### ISSUE THREAD

**Issue by thjaeckle** [scullxbones/akka-persistence-mongo#100]
Hi.
I am using the mentioned ReadJournal query `currentPersistenceIds` for Casbah and have > 200.000 persistenceIds in the Database.
Apparently those > 200.000 ids form a Document > 16MB and I get following Exception:
```
2016-02-18T14:29:19.000+00:00 [App/0] ERR Caused by: com.mongodb.MongoCommandException: Command failed with error 17217: 'exception: distinct too big, 16mb cap' on server mdb001uans1.clouduser.bicadmin.com:30000. The full response is { "errmsg" : "exception: distinct too big, 16mb cap", "code" : 17217, "ok" : 0.0 }
2016-02-18T14:29:19.000+00:00 [App/0] ERR at com.mongodb.connection.ProtocolHelper.getCommandFailureException(ProtocolHelper.java:77) ~[app/:na]
2016-02-18T14:29:19.000+00:00 [App/0] ERR at com.mongodb.connection.CommandProtocol.createCommandResult(CommandProtocol.java:140) ~[app/:na]
2016-02-18T14:29:19.000+00:00 [App/0] ERR at com.mongodb.connection.CommandProtocol.receiveMessage(CommandProtocol.java:131) ~[app/:na]
2016-02-18T14:29:19.000+00:00 [App/0] ERR at com.mongodb.connection.CommandProtocol.execute(CommandProtocol.java:84) ~[app/:na]
2016-02-18T14:29:19.000+00:00 [App/0] ERR at com.mongodb.connection.DefaultServer$DefaultServerProtocolExecutor.execute(DefaultServer.java:155) ~[app/:na]
2016-02-18T14:29:19.000+00:00 [App/0] ERR at com.mongodb.connection.DefaultServerConnection.executeProtocol(DefaultServerConnection.java:219) ~[app/:na]
```

**Comment by scullxbones** [comment]
Oh.  Wow.  Ok.

Let me think about this, but my first guess is I'll have to start tracking `persistenceId`s in a separate collection then.

**Comment by scullxbones** [comment]
Or maybe I can use `aggregate` and `$group` together.  I'd prefer not to add yet another collection due to atomicity constraints.

**Comment by scullxbones** [comment]
FYI - The casbah changes were smooth.  Still working through rxmongo.

**Comment by scullxbones** [comment]
Fixed in `1.2.2`
